Sources/SimultaneouslyScrollView/SimultaneouslyScrollViewHandler.swift

```diff
@@ -3,6 +3,17 @@ import Combine
 import UIKit
 #endif
 
+public struct ScrollDirection: OptionSet {
```

Comment:
Same as above: I would prefer moving this to a separate file.

In addition to that, why do you use an `OptionSet` rather than an enum? I do not plan on Objective C support so enums should be the way to go IMO.

Also the `@available` annotations and the `#if os(iOS)` are missing and should be similar to the other public types.

Reply:
The OptionSet allows us to specify multiple values such as `[.vertical, .horizontal]` or single value when needed such as `.vertical` for example.

Reply:
But since there are only a max of two values (`vertical` and `horizontal`), I wonder if the `OptionSet` is not overkill. It's not like there are many combinations which make sense. Also, the `switch` statement indicates this (only single option cases are there)